app/scripts/controllers/about.js

```diff
@@ -12,31 +12,34 @@
    * Controller of the angularFormentryApp
    */
   angular
-            .module('angularFormentry')
-            .controller('AboutCtrl', AboutCtrl);
+    .module('angularFormentry')
+    .controller('AboutCtrl', AboutCtrl);
 
   AboutCtrl.$inject = ['$log', '$location', '$scope',
-        'FormEntry', '$timeout', '$filter',
-        'TestService', 'UtilService'
-    ];
+    'FormEntry', '$timeout', '$filter',
+    'TestService', 'UtilService'
+  ];
 
   function AboutCtrl($log, $location, $scope, FormEntry,
-            $timeout, $filter, TestService, UtilService, configService) {
+    $timeout, $filter, TestService, UtilService, configService) {
     $scope.vm = {};
     $scope.vm.model = {};
+    $scope.vm.questionMap = {};
+    $scope.vm.hasClickedSubmit = false;
     var schema;
     var newForm;
-    var testSchema = 'triage';
```

Comment:
I would have loved to maintain this variable that people can have the freedom of changing the form without affecting the method